### PR TITLE
** DO NOT MERGE **  Valet key docs update

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

**Comment by PRMerger8** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/patterns/valet-key.md [hunk 1]
```diff
@@ -112,19 +112,26 @@ Azure shared access signatures also support server-stored access policies that c
...
 ```csharp
-public class ValuesController : ApiController
+[ApiController]
+public class SasController : ControllerBase
 {
-  private readonly BlobServiceClient blobServiceClient;
-  private readonly string blobContainer;
+   private readonly string blobContainer = "valetkeysample";
+   private readonly string blobEndpoint;
+   private readonly IConfiguration configuration;
```
Comment (ckittel) [inline docs/patterns/valet-key.md:120]: I'd remove this line as it isn't shown in the sample
```suggestion
```

File: docs/patterns/valet-key.md [hunk 2]
```diff
@@ -136,12 +143,16 @@ public class ValuesController : ApiController
...
   
+      var storageSharedKeyCredential = new StorageSharedKeyCredential(blobServiceClient.AccountName, key.Value);
+      var sas = blobSasBuilder.ToSasQueryParameters(storageSharedKeyCredential).ToString();
+
       return new StorageEntitySas
       {
-          BlobUri = blob.Uri,
-          Credentials = blobSasBuilder.ToSasQueryParameters(storageSharedKeyCredential).ToString()
+          BlobUri = blobClient.Uri,
+          Credentials = sas
```
Comment (ckittel) [inline docs/patterns/valet-key.md:152]:
```suggestion
          Credentials = blobSasBuilder.ToSasQueryParameters(storageSharedKeyCredential).ToString()
```

And then remove line 147.  It's okay if it's a bit different than the impl.  Effectively the same.

**Review by ckittel — CHANGES_REQUESTED** [review]
Just two quick recommendations.

**Comment by ckittel** [comment]
@magrande -- looks like there are merge conflicts as well.  Did you use a stale branch to do this update?  Please resolve the conflict as well.

**Comment by PRMerger13** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger5** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ckittel** [comment]
Moved to `-pr` repo @ MicrosoftDocs/architecture-center-pr#5088

**Pull request closed without merging**